src/webportal/src/app/job/job-view/fabric/job-detail/components/clone-button.jsx

```diff
@@ -28,7 +28,7 @@ const CloneButton = ({ rawJobConfig, namespace, jobName }) => {
       op: 'resubmit',
       type: 'job',
       user: namespace,
-      jobName: jobName,
+      jobname: jobName,
```

Comment:
Seems in PAI codebase, there are still many places using jobName

Reply:
close this pr for further alignment